doc/md_src_files/090_futureWork.md

```diff
@@ -2,16 +2,56 @@
+For all but the most dire vulnerabilities, what the stakeholder chooses to do may include accepting the vulnerability risk because the change risk or other costs of mitigation or remediation are too high.
+Future work should attempt to provide a method for [evaluating change risk or cost](https://github.com/CERTCC/SSVC/issues/35) relative to vulnerability risk.
+
+[Tree Construction and Customization Guidance](#tree-construction-and-customization-guidance) discusses how the prioritization labels in an SSVC tree reflect risk appetite or risk tolerance.
+Specifically, these reflect vulnerability risk appetite.
+Appetite for vulnerability risk may be negatively correlated with change risk; future work could explore this relationship.
+Furthermore, future work could examine suggested practices for connecting tree customization to risk management.
+
+[Reasoning Steps Forward](#reasoning-steps-forward) states the scope of SSVC analysis is "consider credible effects based on known use cases of the software system as a part of cyber-physical systems."
+The unit of prioritization in SSVC should be work items.
+For deployers, a work item is often applying a patch that addresses multiple vulnerabilities.
+The "credible effects" to consider are those of all vulnerabilities remediated by the patch.
+How exactly to aggregate these different effects is not currently specified except to say that the unit of analysis is the whole work item.
+Future work should provide some examples of how this holistic analysis of multiple vulnerabilities remediated in one patch should be conducted.
+
+
 ## Further Decision Tree Testing
 
 More testing with diverse analysts is necessary before the decision trees are reliable. In this context, **reliable** means that two analysts, given the same vulnerability description and decision process description, will reach the same decision. Such reliability is important if scores and priorities are going to be useful. If they are not reliable, they will vary widely over time and among analysts. Such variability makes it impossible to tell whether a difference in scores is really due to one vulnerability being higher priority than other.
 
-The pilot study provides a methodology for measuring and evaluating reliability of the decision process description based on the agreement measure κ. This study methodology should be repeated with different analyst groups, from different sectors and with different experience, feeding the results into changes in the decision process description until the agreement measure is adequately close to 1.
+The SSVC v1 pilot study provides a methodology for measuring and evaluating reliability of the decision process description based on the agreement measure κ.
+This study methodology should be repeated with different analyst groups, from different sectors and with different experience, feeding the results into changes in the decision process description until the agreement measure is adequately close to 1.
 
```

Comment:
The SSVC v1 pilot study provides a methodology...[ ] This study methodology should be repeated with different analyst groups, from different sectors and with different experience, feeding the results into changes in the decision process description until the agreement measure is adequately close to 1.
-- sounds like we're controlling the testing to get the results we expect/want --

Suggested change: The SSVC v1 pilot study provided a methodology...[ ] We are looking for different analyst groups, from different sectors and with different experience,  to further test this methodology.  The testing would be feeding the results into changes in the decision process description to determine the agreement measure. Is the agreement measures adequately close to 1?  -- is this something we should do? --

Reply:
I'm not sure I understand, both of these read the same to me. Both describe a feedback loop between a measurement and improving the system (SSVC). Sorry that I'm not understanding. 

Maybe "using the results to make changes" would be better than "feeding the results into changes" and that would do it?

Reply:
My first question: Are we going to do any testing? E.g., sending out detailed use cases and ask for the results and analysis supporting the results? If not, maybe change it to a theoretical study.  Or we could plan on doing a study in preparation for V3. 

Yes, at a minimum, change it to "using the results to make changes".  'Feeding the results' sounds like manipulation. <shrug>

Reply:
The plan is to do testing in the same way that 070 describes for v1, but on a wider set of people. We will do that once the v2.0 doc is settled, though, so we have something to test.